Implementation/Services/index.md

```diff
@@ -597,3 +597,37 @@ To access the service directly from the view you would need to use the Razor `@i
     <div class="container">
         <article>
 ```
+
+### Handle routes as server-side requests
+Sometimes you want to request for example "/sitemap.xml" from your server, but as this has a file extension, this will be treated as a client side request,
```

Comment:
Suggestion: Break the sentence into smaller sentences as -

Sometimes you might want to request, for example "/sitemap.xml" from your server, but since this has a file extension it will be treated as a client-side request and will not work. You can configure routes to be handled as server-side requests in your `startup.cs`.